bioio_ome_zarr/reader.py

```diff
@@ -99,14 +98,27 @@ def scenes(self) -> Tuple[str, ...]:
                 )
         return self._scenes
 
+    @property
+    def resolution_levels(self) -> Tuple[int, ...]:
+        """
+        Returns
+        -------
+        resolution_levels: Tuple[str, ...]
+            Return the available resolution levels for the current scene.
+            By default these are ordered from highest resolution to lowest
+            resolution.
+        """
+        return tuple(rl for rl in range(len(self._zarr.root_attrs["multiscales"][self.current_scene_index]["datasets"])))
+
     def _read_delayed(self) -> xr.DataArray:
         return self._xarr_format(delayed=True)
 
     def _read_immediate(self) -> xr.DataArray:
         return self._xarr_format(delayed=False)
 
     def _xarr_format(self, delayed: bool) -> xr.DataArray:
-        image_data = self._zarr.load(str(self.current_scene_index))
+        data_path = self._zarr.root_attrs["multiscales"][self.current_scene_index]["datasets"][self.current_resolution_level]["path"]
```

Comment:
Nit: maybe check that the level is valid before indexing into datasets here?

Reply:
Hmm do you think that is necessary with `set_current_resolution_level()` checking to see if it is valid at set time?